rust/perspective-python/build.mjs

```diff
@@ -37,6 +37,7 @@ const version = pkg.version.replace(/-(rc|alpha|beta)\.\d+/, (x) =>
 );
 
 fs.mkdirSync(`./perspective_python-${version}.data`, { recursive: true });
+fs.copyFileSync("../../LICENSE.md", "./LICENSE.md");
```

Comment:
it turns out `maturin develop` needs this file around now that it's specified in `pyproject.toml` as `{license { file = "LICENSE.md" }}`, so I moved it out of the sdist branch so it always runs.